.github/workflows/api-release.yml

```
@@ -39,0 +1,81 @@
+          GPG_PRIVATE_KEY: ${{ secrets.gpg_private_key }}
+          SIGN_KEY_PASS: ${{ secrets.gpg_passphrase }}
+          CENTRAL_USERNAME: ${{ secrets.sonatype_username }}
+          CENTRAL_PASSWORD: ${{ secrets.sonatype_password }}
+
+  build-image:
+    if: ${{ inputs.module != 'handler-core' }}
+    needs: release-maven
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build and push image
+        uses: it-at-m/lhm_actions/action-templates/actions/action-build-image@b2b49a8b4344e3b5ac90dd1ee9a0c59860c41383 # v1.0.16
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)